engine/proto/master.proto

```diff
@@ -109,8 +109,8 @@ service JobManager {
 message Selector {
     enum Op {
         OpUnknown = 0;
-        Eq = 1;
-        Neq = 2;
+        Equal = 1;
+        NotEqual = 2;
```

Comment:
`Eq` and `Neq` seems fine to me.

Reply:
I prefer `Eq` and `Neq`, which are more simple and they are also commonly used